build/includes/release.mk

```diff
@@ -52,15 +52,16 @@ release-deploy-site:
 	echo "Deploying Site Version: $$version" && \
 	$(MAKE) ENV=HUGO_ENV=snapshot site-deploy SERVICE=$$version
 
-# no-op-release
-void-release: RELEASE_VERSION ?= $(base_version)
-void-release: $(ensure-build-image)
+# creates release-branch
+release-branch: RELEASE_VERSION ?= $(base_version)
+release-branch: $(ensure-build-image)
 	@echo "Starting release for version: $(RELEASE_VERSION)"
 
 	# switch to the right project
 	$(DOCKER_RUN) gcloud config configurations activate agones-images
-
-	git checkout -b release-$(RELEASE_VERSION)	
+    git remote update -p
```

Comment:
You may need a tab here to get rid of the "missing separator" error

Reply:
Corrected it by adding a new line